Makefile

```diff
@@ -129,7 +129,7 @@ run: generate fmt vet manifests ## Run against the configured Kubernetes cluster
 	OPERATOR_POD_NAME=$(OPERATOR_POD_NAME) \
 	OPERATOR_NAMESPACE=$(OPERATOR_NAMESPACE) \
 	go run ./cmd/manager/main.go --atlas-domain=$(ATLAS_DOMAIN) \
-	--global-api-secret-name=$(ATLAS_KEY_SECRET_NAME)
+	--global-api-secret-name=$(ATLAS_KEY_SECRET_NAME) --log-level=debug
```

Comment:
Do we want this debug log level all the time here or only during current development?

Reply:
I don't see a case where someone would start the manager manually other than in development, and in that case, we usually are interested in debug-level messages.